personalization-webcomponents/src/components/ChecklistHeader.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div
+      class="m-intro m-intro-static-image"
+      style="background-color: var(--color-neutrals-blue-xlight)"
+  >
+    <div class="container">
+      <div style="display: grid; grid-template-columns: 75px auto; column-gap: 20px">
```

Comment:
I would prefer to use the classes provided by patternlab here: https://patternlab.muenchen.space/?p=components-intro-vertical-contact